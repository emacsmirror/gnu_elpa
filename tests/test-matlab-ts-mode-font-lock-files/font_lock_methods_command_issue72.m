% -*- matlab-ts -*-

% See https://github.com/acristoffers/tree-sitter-matlab/issues/72

m = methods('foo')

methods foo

