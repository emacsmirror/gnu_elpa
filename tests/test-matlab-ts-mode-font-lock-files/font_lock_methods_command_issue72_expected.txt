c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

v o kkkkkkkbSsssSb

FFFFFFF aaa

